recipes/casadi/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "casadi" %}
+{% set version = "3.1.1" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/casadi/casadi.git
+  git_rev: {{ version }}
```

Comment:
Please change this to a tarball like https://github.com/ocefpaf/libdap4-feedstock/blob/master/recipe/meta.yaml#L7-L10 for example.